actors/test/actor-tests/src/test/java/cloud/orbit/actors/test/DeactivationTest.java

```diff
@@ -63,6 +63,7 @@ public void simpleCleanupTest() throws ExecutionException, InterruptedException
         final UUID id = actor1.getUniqueActivationId().join();
         clock.incrementTime(20, TimeUnit.MINUTES);
         stage.cleanup().join();
+        waitForDeactivations();
```

Comment:
I disagree, cleanup explicitly invokes await(localObjectsCleaner.cleanup()); and we are joining on the cf returned by cleanup so I suspect there is something wrong with the new impl and waitForDeactivations feels like a hack.